docs/components/movement-sensor/_index.md

```diff
@@ -42,8 +42,9 @@ Model | Description <a name="model-table"></a>
 
 ## Control your movement sensor with Viam's client SDK libraries
 
-To get started using Viam's SDKs to connect to and control your robot, go to your robot's page on [the Viam app](https://app.viam.com) and navigate to the **Code sample** tab.
-Then, select your preferred programming language, toggle **Include secret** to display your robot's location secret in the code, and copy the sample code generated.
+To get started using Viam's SDKs to connect to and control your robot, go to your robot's page on [the Viam app](https://app.viam.com), navigate to the **Code sample** tab, select your preferred programming language, and copy the sample code generated.
+
+{{% snippet "show-secret.md" %}}

```

Comment:
this is fine - but worth pointing out that you could've also included the previous line in the snippet